test/spec/estimatedScore/estimatedScoreEventsHandlerSrv.spec.js

```diff
@@ -84,10 +86,7 @@ describe('testing service "EstimatedScoreEventsHandlerSrv":', function () {
         var expectedResult = {
             exerciseType: ExerciseTypeEnum.SECTION.enum,
             exerciseId: section.id,
-            score: (0 * 90) + (1 * 100) + (2 * 120) + (1 * 140) + (1 * 150) + //correct
-            (1 * 50) + (3 * 60) + (2 * 80) + (1 * 100) + (2 * 120) //wrong
-            //score: (/*3*/ 3 * 120) + (/*2*/ 1 * 100) + (/*4*/ 1 * 140) +
-            //(/*1*/ 3 * 50) + (/*5*/ 2 * 120) + (/*4*/ 3 * 100) + (/*2*/ 2 * 60) + (/*3*/ 2 * 80)
+            score: 1360
```

Comment:
why this is the score ? 
i prefer to see how it was calculated as it was before